go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/diktyo-io/networktopology-api v0.0.8-alpha
 	github.com/dustin/go-humanize v1.0.0
 	github.com/google/go-cmp v0.5.7
-	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.12
+	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.13
```

Comment:
please mention in the commit message that you want to consume the new policies (https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/pull/23) )

Reply:
done